src/Evaluator.ts

```diff
@@ -125,14 +125,14 @@ export class Evaluator {
   private recomputeFormulas(cycled: Vertex[], sorted: Vertex[]): void {
     cycled.forEach((vertex: Vertex) => {
       if (vertex instanceof FormulaVertex) {
-        vertex.setCellValue(new CellError(ErrorType.CYCLE, undefined, vertex.getAddress(this.lazilyTransformingAstService)))
+        vertex.setCellValue(new CellError(ErrorType.CYCLE, undefined, vertex))
```

Comment:
cute, since this is such a natural thing to do, this hints that this is a proper way of actually approaching this issue